application/helpers/admin/import_helper.php

```
@@ -2617,6 +2656,17 @@ function XMLImportSurvey($sFullFilePath, $sXMLdata = null, $sNewSurveyName = nul
 
     $oldNewFieldRoots = sortByKeyLengthDescending($oldNewFieldRoots);
 
+    foreach ($aGIDReplacements as $oldGid => $newGid) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$newGid'. (undefined)

(UnusedLocalVariable)